Fix errors introduced by new flake8

See https://travis-ci.org/pytrip/pytrip/jobs/489773600

```
packaging runtests: commands[1] | flake8 .
./pytrip/paths.py:118:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/paths.py:210:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/paths.py:212:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/vdx.py:1119:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/field.py:128:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/field.py:132:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/execute.py:540:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/execute.py:573:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/plan.py:485:16: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/plan.py:568:12: F632 use ==/!= to compare str, bytes, and int literals
./pytrip/tripexecuter/plan.py:604:21: E117 over-indented
./pytrip/utils/spc2pdf.py:113:34: E231 missing whitespace after ','
```

--- a/pytrip/paths.py
+++ b/pytrip/paths.py
@@ -115,7 +115,7 @@ class DensityProjections:
     def optimize_angle(self, voi, couch, gantry, margin, iteration, avoid=[]):
         """ TODO: Documentation
         """
-        if iteration is 0:
+        if iteration == 0:
             return [gantry, couch]
         grid = self.calculate_quality_list(
             voi, np.linspace(gantry, gantry + margin, 3), np.linspace(couch, couch + margin, 3), avoid=avoid)
@@ -207,9 +207,9 @@ class DensityProjections:
         dimension = window_size / self.cube.pixel_size
         dimension = np.int16(dimension)
         start = center - self.cube.pixel_size * 0.5 * np.array(dimension[0] * b + dimension[1] * c)
-        if calculate_from is 1:
+        if calculate_from == 1:
             start = self.calculate_back_start_voi(voi, start, step_vec)
-        if calculate_from is 2:
+        elif calculate_from == 2:
             start = self.calculate_front_start_voi(voi, start, step_vec)
 
         data = pytriplib.calculate_wepl(

--- a/pytrip/tripexecuter/execute.py
+++ b/pytrip/tripexecuter/execute.py
@@ -537,7 +537,7 @@ class Execute(object):
         ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
 
         # If no password is supplied, try to look for a private key
-        if self.password is "" or None:
+        if not self.password:
             rsa_keypath = os.path.expanduser(self.rsakey_local_path)
             if not os.path.isfile(rsa_keypath):
                 # login with provided username + empty password
@@ -570,7 +570,7 @@ class Execute(object):
         # transport.set_missing_host_key_policy(paramiko.AutoAddPolicy())
 
         # If no password is supplied, try to look for a private key
-        if self.password is "" or None:
+        if not self.password:
             rsa_keypath = os.path.expanduser(self.rsakey_local_path)
             if not os.path.isfile(rsa_keypath):
                 # login with provided username + empty password

--- a/pytrip/tripexecuter/field.py
+++ b/pytrip/tripexecuter/field.py
@@ -125,11 +125,11 @@ class Field(object):
         from the target Voi provided.
 
         """
-        if len(isocenter_str) is 0:
+        if len(isocenter_str) == 0:
             self.isocenter = []
             return
         _target = isocenter_str.split(",")
-        if len(_target) is 3:
+        if len(_target) == 3:
             try:
                 self.isocenter = [float(_target[0]), float(_target[1]), float(_target[2])]
                 return

--- a/pytrip/tripexecuter/plan.py
+++ b/pytrip/tripexecuter/plan.py
@@ -482,10 +482,10 @@ class Plan(object):
 
             # set isocenter if specified in field
             _ic = _field.isocenter
-            if len(_ic) is not 0:
+            if len(_ic) >= 3:
                 line += " target({:.1f},{:.1f},{:.1f}) ".format(_ic[0], _ic[1], _ic[2])
 
-            # set dose extention:
+            # set dose extension:
             # TODO: check number of decimals which make sense
             # TODO: zeros allowed?
             line += " doseext({:.4f})".format(_field.dose_extension)
@@ -565,7 +565,7 @@ class Plan(object):
         output = []
         window = self.window
         window_str = ""
-        if len(window) is 6:
+        if len(window) == 6:
             window_str = " window({:.2f},{:.2f},{:.2f},{:.2f},{:.2f},{:.2f} ".format(window[0], window[1],  # Xmin/max
                                                                                      window[2], window[3],  # Ymin/max
                                                                                      window[4], window[5])  # Zmin/max
@@ -601,8 +601,8 @@ class Plan(object):
 
         if self.want_rst and self.optimize:
             for i, field in enumerate(fields):
-                    output.append('field {:d} / write file({:s}.rst) reverseorder '.format(i + 1, field.basename))
-                    self._out_files.append(field.basename + ".rst")
+                output.append('field {:d} / write file({:s}.rst) reverseorder '.format(i + 1, field.basename))
+                self._out_files.append(field.basename + ".rst")
 
         # TODO: add various .gd files
         return output

--- a/pytrip/vdx.py
+++ b/pytrip/vdx.py
@@ -1116,7 +1116,7 @@ class Voi:
 
         :returns: a list of str holding the all lines needed for a Voxelplan formatted file.
         """
-        if len(self.slices) is 0:
+        if len(self.slices) == 0:
             return ""
 
         out = "\n"

